src/docs/introduction/architecture.md

```diff
@@ -1 +1,30 @@
-# Architecture
\ No newline at end of file
+# Architecture
+
+Sealer has two top module: Image Module & Cluster Module
+
+The Image module using Kubefile and build context as input, and build a ClusterImage that contains all the dependencies.
+The Cluster module using Clusterfile to init a cluster which contains kubernetes and other applications.
```

Comment:
For Cluster module, Sealer Image is the main object to run, and the Clusterfile is only a config file